src/myr/Myr.js

```diff
@@ -670,7 +680,64 @@ class Myr {
     setTextureColoring = (i) => {
         this.cursor.textureColoring = Boolean(i);
         return this.cursor.textureColoring;
-    };
+    }
+
+    /**
+     * Changes the material type of the cursor.
+     * 
+     * @param {string} materialtype 
+     * @returns 
+     */
+    setMaterialType = (materialtype = MaterialType.SPECULAR) => {
+        let validMaterial = false;
+        for(let key in MaterialType) {
+            validMaterial = MaterialType[key] === materialtype;
+            if(validMaterial) break;
+        }
+        if(!validMaterial) {
+            throw new Error(`${materialtype} is not a valid material type!`);
+        }
+
+        this.cursor.materialtype = materialtype;
+        return this.cursor.materialtype;
+    }
```

Comment:
If one doesn't exist there should be a "getMaterialType()" which should also be added to the reference, autocomplete, etc.